src/messaging/tests/echo/README.md

```diff
@@ -0,0 +1,52 @@
+# CHIP Example Application Tutorial
+
+## Introduction
+
+The CHIP Echo example application shows you how to implement a CHIP application
+program using one of the supported CHIP protocols, namely the very simple CHIP
+Echo protocol. This protocol allows you to send a CHIP message to a peer and
+expect a CHIP response (similar to the ICMP Echo Request/Echo Response
+messages).
+
+CHIP Protocols are, essentially, implementations of specific protocols over the
+CHIP transport. Furthermore, when two CHIP nodes are exchanging messages of a
+particular CHIP protocol, they do so over a construct called a CHIP Exchange
+which is a description of a CHIP-based conversation over a CHIP protocol. A CHIP
+Exchange is characterised by the ExchangeContext object, and every CHIP node
+must create an ExchangeContext object before initiating a CHIP conversation.
+
+After constructing a CHIP ExchangeContext, CHIP messages are sent and received
+using the ChipMessageLayer class which sends the CHIP message over a chosen
+transport (TCP, UDP, or CRMP).
+
+## Building
+
+```
+source scripts/activate.sh
+cd src/messaging/tests/echo
```

Comment:
Suggest removing this line, there is no difference building in src/messaging/tests/echo vs building at the top level so this may be confusing or waste disk space.

Reply:
Acked